test/dummy-sorcery/db/migrate/20220825200321_populate_test_users.rb

```diff
@@ -1,17 +1,17 @@
 class PopulateTestUsers < ActiveRecord::Migration[6.0]
   def change
     add_column :users, :role, :string
-    100.times { User.create(name: gen_name, age: rand(100), email: gen_email, role: gen_role, password: '12345678', password_confirmation: '12345678') }
+    100.times { |t| User.create(name: gen_name(t), age: rand(100), email: gen_email(t), role: gen_role, password: '12345678') }
   end
 
   private
 
-  def gen_name
-    ['Igor', 'Michael' 'John', 'Stan', 'Bob', 'Kris', 'Alan'].sample + ' ' + rand(100).to_s
+  def gen_name(number)
+    ['Igor', 'Michael' 'John', 'Stan', 'Bob', 'Kris', 'Alan'].sample + ' ' + number.to_s
```

Comment:
For some reason locally I was getting a LOT of random collisions when trying to migrate. By making this consistent it took care of the collisions.